assets/src/components/v2/pre_fare_single_screen_alert.tsx

```diff
@@ -266,13 +272,13 @@ const PreFareAlertBanner: React.ComponentType<{routes: any[]}> = ({routes}) => {
     banner = <div className={classWithModifiers("alert-banner", ["large--two-routes", getAlertColor(routes)])}>
       <span><span className="alert-banner__attention-text">ATTENTION,</span> riders to</span>
       {routes.map((route) => {
-        const LinePill = STRING_TO_SVG[route]
-        return <LinePill className="alert-banner__route-pill--long" color={getHexColor(getRouteColor(route))} />
+        const LinePill = STRING_TO_SVG[route.svg_name]
+        return <LinePill className="alert-banner__route-pill--long" color={getHexColor(getRouteColor(route.route_id))} />
```

Comment:
```suggestion
        return <LinePill className="alert-banner__route-pill--long" key={route.svg_name} color={getHexColor(getRouteColor(route.route_id))} />
```

This is the change I mentioned in the other PR. Nothing wrong visually, but will help get rid of a console error.

Reply:
Thank you!